tests/test_namespace.py

```diff
@@ -515,6 +515,16 @@ def test_get_entity_name(self):
             "City_T"
         )
 
+    def test_autocompletion_ipython(self):
+        """Checks that all the expected ontology entities are in __dir__.
+
+        The check is done just for the `cuba` namespace.
+        """
+        expected = {'activeRelationship', 'passiveRelationship',
+                    'relationship', 'attribute', 'path', 'Entity', 'File',
+                    'Nothing', 'Wrapper'}
+        self.assertSetEqual(set(cuba.__dir__()) & expected, expected)
```

Comment:
Here I would call `dir(cuba)`, which is what the magic function is acting for.

Reply:
Done, thanks!